artichoke-backend/src/extn/core/kernel/trampoline.rs

```diff
@@ -54,6 +54,20 @@ pub fn puts(interp: &mut Artichoke, args: Vec<Value>) -> Result<Value, Exception
     Ok(interp.convert(None::<Value>))
 }
 
+pub fn p(interp: &mut Artichoke, args: Vec<Value>) -> Result<Value, Exception> {
```

Comment:
I merged a change earlier today to have `print` and `puts` take `IntoIterator` arguments so they can be used on iterators in addition to `Vec`s. Could you change the signature to something like this?

https://github.com/artichoke/artichoke/blob/3ba3ece1939f2dee96c9442a91cbf7a6f79acc1d/artichoke-backend/src/extn/core/kernel/trampoline.rs#L21-L23

This shouldn't require any changes to the `mruby` glue.